### DISCUSSION TITLE
Should we enforce decimals for whole numbers in our calculators?

### DISCUSSION THREAD

**Discussion by jeffrey-tafe** [gh44922023/Tafe230221#4] [General]
As above

**Comment by BrodieBrown1998** [comment]
Both, you should use what the section calls for.

**Reply by jeffrey-tafe** [reply]
I think it would be best to cast all inputs and outputs as doubles by default. Not a requirement, but just an idea. Maybe an enhancement for the release after this one. @gh44922023 What do you think?

**Comment by gh44922023** [comment]
> I think you raise a valid point and something for future releases and project consistencies.
